scripts/monitor/rano_monitor/widgets/summary.py

```diff
@@ -80,5 +79,15 @@ def update_summary(self):
 
     def on_button_pressed(self, event: Button.Pressed) -> None:
         event.stop()
-        package_review_cases(self.report, self.dset_path)
-        self.notify(f"{REVIEW_FILENAME} was created on the working directory")
+        pkg_btn = self.query_one("#package-btn", Button)
+        unpkg_btn = self.query_one("#unpackage-btn", Button)
```

Comment:
just double-checking id #package-btn should be #package-btns. (Just hunting for typos :))

Reply:
Thanks for the check! In this case it shouldn't be, as that would be the container of both buttons, and not the specific button